projects/mongo-core-test.yml

```diff
@@ -0,0 +1,26 @@
+---
+runner: command
+params:
+  dir: mongo-core-test
+  setup: |
```

Comment:
Is this sensible why to setup and run tests? We probably should have `cleanup` to stop the container.

Reply:
Yeah, let's add `teardown` in a separate PR

Reply:
Let's add teardown now

Reply:
Added